src/components/pager/js/Pager.js

```diff
@@ -514,6 +513,8 @@ var fluid_2_0 = fluid_2_0 || {};
         },
         invokers: {
             acquireDefaultRange: {
+                // TODO: problem here - these are dynamic components and so cannot be constructed gingerly
```

Comment:
Which "these" is this comment referring to? What does the expression "current (old) framework" refer to specifically?